### PR TITLE
Add TD Consumption step

### LINKED ISSUE #231: Explain TD validation containing protocol binding
Similar to https://w3c.github.io/wot-thing-description/#validation-serialization-json , we should explain what it means to validate a TD containing bindings.

### PR DESCRIPTION
**Pull request by egekorkan**
fixes #231 

also from https://github.com/w3c/wot-binding-templates/pull/263#issuecomment-1470244570, I have started a very initial work on "We need to explain how Consumers should identify the protocol from the URI.".

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-binding-templates/pull/268.html" title="Last updated on Mar 29, 2023, 2:51 PM UTC (41eb989)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-binding-templates/268/c314cbd...41eb989.html" title="Last updated on Mar 29, 2023, 2:51 PM UTC (41eb989)">Diff</a>

### CODE REVIEW

**Review by relu91 — APPROVED** [review]
In scripting API consuming a TD is only about validating. Any other protocol-related action (including initialization) is done only upon interaction (on-demand). I see this text is not aligned with what we are saying in the API, but we can improve it later. Also, the introduction text needs improvements. 

Anyhow, I'm ok for tackling this issues in a future PR.

File: index.html [hunk 1]
```diff
@@ -448,6 +448,34 @@ <h4>Existing Protocol Binding Templates</h4>
...
+
+                <p>
+                    Protocol Binding Templates contain vocabularies that extend the vocabulary found in the [[WOT-THING-DESCRIPTION]].
+                    This means that the way a TD is consumed is adapted to such vocabularies.
+                    The steps below explain how this process typically looks like.
+                    <!-- this text should be extended after the PR 262 is merged -->
+                </p>
+                <ol>
+                    <li>
+                        <b>Detect the protocol:</b> Upon the activation of a form to execute the operation, the Consumer
```
[the remark this answers is not in the document]
Comment (egekorkan) [reply index.html:462]: I can do this :)

**Comment by egekorkan** [comment]
> In scripting API consuming a TD is only about validating

I will add to the title "and interacting".

**Comment by egekorkan** [comment]
Call of 22.03:
- Relaxing the JSON Schema to say "you may use that" but other methods are possible
- Detecting the protocol should be clear that **for now** is from the URI but other mechanisms might be possible in the future.
- Add a note about where to find the data schema for the operation
- Data Schema and Content Type should be linked to the definitions
- In the next charter, it can go into another place (like TD)
- This is a bit of a "technical" issue compared to other ones, should be identifiable.

File: index.html [hunk 2]
```diff
@@ -521,6 +523,35 @@ <h4>Existing Protocol Binding Templates</h4>
                     </tbody>
                 </table>
             </section>
+            <section>
+                <h4>Using a TD with a Protocol Binding Template Subspecification</h4>
```
Comment (egekorkan) [inline index.html:527]:
```suggestion
                <h4>Using a Thing Description with a Protocol Binding Template Subspecification</h4>
```

**Comment by egekorkan** [comment]
Call of 29.03:
- We decided not use acronyms in titles
- the naming with regards to Protocol Binding _Template_ should be discussed later on.
- merging

**Pull request merged**